app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectInformationActivity.kt

```diff
@@ -439,6 +457,31 @@ class ProjectInformationActivity : AppCompatActivity() {
         }, { showToast(getString(R.string.could_not_download_video), Toast.LENGTH_LONG) })
     }
 
+    /**
+     * moves a video from the delete Directory to the copy Directory
+     *
+     * @param fileUrl url of the file to be moved
+     * @param deleteDirectory directory from which it has to be moved
+     * @param copyDirectory directory in which it is moved to
+     */
+    private fun movingVideo(fileUrl: String, deleteDirectory: File, copyDirectory: File){
+        val fileName = fileDB.getFileName(fileUrl)
+        val file = File(deleteDirectory, fileName)
+        if(file.exists()){
+            val newFile = File(copyDirectory, fileName)
+            try{
+                Files.move(file, newFile)
+            }catch(e: Exception){
+                
+            }
```

Comment:
Catching any `Exception` is usually a bad practice, potentially **hiding a deeper** issue in the code-base.
Please catch an `Exception` as precise as it is possible.
(Also you can show a toast message showing the error if needed)